assets/sass/adminbar.scss

```
@@ -47,6 +47,7 @@
 @import "components/global/googlesitekit-change-arrow";
 @import "components/global/googlesitekit-cta";
 @import "components/global/googlesitekit-cta-link";
+@import "components/global/googlesitekit-zero-data";
```

Ideally we wouldn't need to add a new stylesheet/module for this but if we do, it should also be renamed.
```suggestion
@import "components/global/googlesitekit-admin-bar-zero-data";
```